config/database.yml

```diff
@@ -1,7 +1,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: <%= ENV.fetch 'RAILS_MAX_THREADS', 10 %>
```

Comment:
You set the pool to 10, but max_threads_count is still 5. Is it because there are 2 workers?

Reply:
> If you are using the Puma web server we recommend setting the pool value to equal ENV['RAILS_MAX_THREADS']. When using multiple processes each process will contain its own pool so as long as no worker process has more than ENV['RAILS_MAX_THREADS'] then this setting should be adequate.

https://devcenter.heroku.com/articles/concurrency-and-database-connections